packages/create-twilio-function/src/create-twilio-function/versions.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  twilio: '^3.56',
```

Comment:
Do we want to be explicit here? Or should we just open it up to `^3` or even `*`? I think even if we put an explicit number here people might still get confused.

Reply:
Given that we are placing this into a package.json for a project, leaving it as `*` or `^3` will mean that in the case of `twilio` version 4 being released a fresh `npm install` in that project will pick the new version and potentially cause breaking changes.

I think that `^3.56` is the right level of semver specificity for `create-twilio-function` to install the latest version of `twilio` into a project until we need to change it to version 4.